CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -509,9 +509,11 @@ function Patient:goHome(reason, disease_id)
 
   hosp:updatePercentages()
 
+  -- Debug patients must be removed from the debug list so they can despawn properly
```

Comment:
Why do we still have this here? They must be removed from hospital somewhere or else we'd leak patients.

Reply:
Not removing them here means they can still be commanded by the player. This interrupts their despawn cycle, and remain stuck in the hospital.